src/components/Nav/Nav.tsx

```diff
@@ -0,0 +1,76 @@
+import classNames from 'classnames';
+import { useContext, useState } from 'react';
+import { SortParam } from '../../types/SortParam';
+import { NavContext } from '../NavContext';
+
+export const Nav: React.FC = () => {
+  const [activeButton, setActiveButton] = useState('all');
+  const { setSortBy } = useContext(NavContext);
+
+  function selectSort(param?: string) {
+    switch (param) {
+      case 'FilterLinkAll':
```

Comment:
would be great to create enum for FilterLinkAll, FilterLinkActive, FilterLinkCompleted